### ISSUE TITLE
NPE - 49.12 MekHQ Meklab - When changing engine size/changing equipment specifically on an ENF-4R

### ISSUE THREAD

**Issue by UlyssesSockdrawer** [MegaMek/mekhq#3654]
### Environment
Version: MHQ 49.12
OS: Win 11
Java: 11

### Description
When changing the engine type (fusion -> XL or vice versa) on the ENF-4R the MekHQ integrated MekLab throws an NPE:

![image](https://user-images.githubusercontent.com/88290873/235310055-c9718cd6-d7be-40ac-8478-8afc45e074f3.png)

I have only had this happen for the ENF-4R and not for any other 'mechs so far.

MekHQ logs attached with multiple NPEs generated. - [mekhq.log](https://github.com/MegaMek/mekhq/files/11359487/mekhq.log)

### Files
Save game and Customs folder: 
[Tarleton's Legion30640803.cpnx.gz](https://github.com/MegaMek/mekhq/files/11359496/Tarleton.s.Legion30640803.cpnx.gz)

[Customs.zip](https://github.com/MegaMek/mekhq/files/11359497/Customs.zip)

**Comment by HammerGS** [comment]
The news errors in the logs are fixed. 

There appears to be something with the AS conversion. @SJuliez 
```
mekhq.MekHQ.lambda$main$0(MekHQ.java:240) - Uncaught Exception Detected
java.lang.NullPointerException
	at megamek.common.Mounted.getExplosionDamage(Mounted.java:1239)
	at megamek.common.alphaStrike.conversion.ASSpecialAbilityConverter.isExplosive(ASSpecialAbilityConverter.java:485)
	at megamek.common.alphaStrike.conversion.ASSpecialAbilityConverter.processENE(ASSpecialAbilityConverter.java:86)
	at megamek.common.alphaStrike.conversion.ASSpecialAbilityConverter.processAbilities(ASSpecialAbilityConverter.java:78)
	at megamek.common.alphaStrike.conversion.ASConverter.performConversion(ASConverter.java:144)
	at megamek.common.alphaStrike.conversion.ASConverter.convertForMechCache(ASConverter.java:45)
	at megameklab.ui.generalUnit.PreviewTab.refresh(PreviewTab.java:66)
	at mekhq.gui.MekLabTab$MekPanel.refreshPreview(MekLabTab.java:821)
	at megameklab.ui.generalUnit.AbstractEquipmentTab.refreshOtherTabs(AbstractEquipmentTab.java:201)
	at megameklab.ui.generalUnit.AbstractEquipmentTab.lambda$new$0(AbstractEquipmentTab.java:77)
	at java.desktop/javax.swing.table.AbstractTableModel.fireTableChanged(Unknown Source)
	at java.desktop/javax.swing.table.AbstractTableModel.fireTableDataChanged(Unknown Source)
	at megameklab.ui.util.CriticalTableModel.refreshModel(CriticalTableModel.java:95)
	at megameklab.ui.generalUnit.AbstractEquipmentTab.fireTableRefresh(AbstractEquipmentTab.java:192)
	at megameklab.ui.generalUnit.AbstractEquipmentTab.refresh(AbstractEquipmentTab.java:183)
	at mekhq.gui.MekLabTab$MekPanel.refreshEquipment(MekLabTab.java:790)
	at megameklab.ui.mek.BMStructureTab.engineChanged(BMStructureTab.java:833)
	at megameklab.ui.mek.BMChassisView.lambda$actionPerformed$5(BMChassisView.java:782)
```

**Comment by NickAragua** [comment]
Fixed by MM #4372
